backend/src/schema/types/Todo.js

```diff
@@ -0,0 +1,24 @@
+const { gql } = require("apollo-server");
+
+module.exports.TodoType = gql`
+  type Todo {
+    id: ID
+    text: String
+    author: Author
+  }
+
+  type Query {
+    todos: [Todo]
+  }
+
+  type Mutation {
+    delToDo(id: ID): [Todo]
+    addToDo(text: String!, authorName: String!): [Todo]
+    updateToDo(id: ID!, text: String!, authorName: String!): [Todo]
```

Comment:
Why on earth are those mutations returning `[Todo]` and not `Todo`? :thinking:

Reply:
this changes in the future